### PR TITLE
Generated beta models and request builders using Kiota

### PR DESCRIPTION
**Pull request by baywet**
This pull request was automatically created by the GitHub Action, **create pull request**. 

 The commit hash is _c68fbd866a0351d8d853659f5488a9b10dce983d_. 

 **Important** Check for unexpected deletions or changes in this PR.

### CODE REVIEW

File: drive/root/assignsensitivitylabel/assign_sensitivity_label_post_request_body.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        return nil
+    }
+    res["sensitivityLabelId"] = func (n i878a80d2330e89d26896388a3f487eef27b0a0e6c010c493bf80be1452208f91.ParseNode) error {
+        val, err := n.GetStringValue()
+        if err != nil {
+            return err
+        }
+        if val != nil {
+            m.SetSensitivityLabelId(val)
+        }
```
Comment (rkodev) [inline drive/root/assignsensitivitylabel/assign_sensitivity_label_post_request_body.go:68]: @baywet when deserializing primitives we might want to simply it to 

```go
        val, err := n.GetStringValue()
        if err != nil {
            return err
        }
        m.SetSensitivityLabelId(val)
        return nil
```

File: deviceregistrationpolicy/device_registration_policy_request_builder.go [hunk 2]
```diff
@@ -1,130 +0,0 @@
-package deviceregistrationpolicy
```
Comment (rkodev) [inline deviceregistrationpolicy/device_registration_policy_request_builder.go:1]: Question: Do we intend to treat deprecation of entire functionality as a major change?

**Comment by rkodev** [comment]
LGTM

File: deviceregistrationpolicy/device_registration_policy_request_builder.go (hunk 2 above)
Comment (baywet) [reply deviceregistrationpolicy/device_registration_policy_request_builder.go:1]: not under beta, breaking changes are ok, this is what beta APIs are for.

**Review by rkodev — DISMISSED** [review]

**Pull request merged**